gwimages/gwimages.ml

```diff
@@ -0,0 +1,47 @@
+
+let key_of_person base p =
+  let surname = Name.lower (Gwdb.p_surname base p) in
+  let first_name = Name.lower (Gwdb.p_first_name base p) in
+  let occ = string_of_int (Gwdb.get_occ p) in
+  let key = String.concat "." [surname; first_name; occ] in
+  key
```

Comment:
```suggestion
  String.concat "." [surname; first_name; occ]
```

Reply:
Today, portrait files are named first_name.oc.surname.ext
Are we departing from this format?

Reply:
it is not the only way you can reference a portrait file, you can also use an external url. We are exporting these.